### PR TITLE
Gidk 1tf04x new inventory test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/1tf04x

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.adjust-inventory-list-page{
+    h4{
+        font-size: 16px;
+        font-family: 'opensans-semibold';
+    }
+    .custom-datepicker{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.scss:6]: Do format the code

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import { GIDDH_DATE_RANGE_PICKER_RANGES } from '../../../app.constant';
+import * as moment from 'moment/moment';
+import { GIDDH_DATE_FORMAT, GIDDH_NEW_DATE_FORMAT_UI } from '../../../shared/helpers/defaultDateFormat';
+@Component({
+    selector: 'adjust-inventory-list',
+    templateUrl: './adjust-inventory-list.component.html',
+    styleUrls: ['./adjust-inventory-list.component.scss'],
+
+})
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts:13]: Remove empty line here

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        if (value && value.startDate && value.endDate) {
+            this.selectedDateRange = { startDate: moment(value.startDate), endDate: moment(value.endDate) };
+            this.selectedDateRangeUi = moment(value.startDate).format(GIDDH_NEW_DATE_FORMAT_UI) + " - " + moment(value.endDate).format(GIDDH_NEW_DATE_FORMAT_UI);
+            this.fromDate = moment(value.startDate).format(GIDDH_DATE_FORMAT);
+            this.toDate = moment(value.endDate).format(GIDDH_DATE_FORMAT);
+        }
+    }
+
+    /* Aside pane state*/
+    public asideMenuState: string = 'out';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts:80]: Member declaration should be before methods

File: apps/web-giddh/src/app/new-inventory/component/create-group/create-group.component.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
+@import "../../../../assets/css/color.scss";
+
+.create-group-page{
+    padding-top: 15px;
+    .image-group{
+        padding-top: 35px;
+        .grodup-vector{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/create-group/create-group.component.scss:7]: Avoid incorrect spelling

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+                                        <p>
+                                            <span class="stockSku"><label>SKU:
+                                                </label>Item SKU Code</span>
+                                            <span><label>Rate:</label>
+                                            </span>2000
+                                        </p>
+                                    </div>
+                                </div>
+
+                                <div class="d-flex align-items-center" style="align-items: center">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.html:45]: Avoid inline styles

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+        background-color: $lavender;
+        color: $chetwode-blue;
+    }
+
+    &:hover h5 {
+        color: $chetwode-blue;
+    }
+}
+
+.stockName {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss:33]: Avoid camelCase class names

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+    border-right: 1px solid $gainsboro;
+    padding-right: 15px;
+}
+
+.add-bulk-item-modal .modal-title {
+    font-size: 22px !important;
+    color: $white;
+}
+
+.table-custom-invoice .salesTax .dropdown-menu-right li label {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss:123]: Avoid camelCase class names `salesTax `

File: apps/web-giddh/src/app/new-inventory/component/inventory-group-list-sidebar/inventory-group-list-sidebar.component.ts [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+    public openSubSidebar(){
+        this.subSidebarOpen = !this.subSidebarOpen;
+        if (document.getElementsByClassName("right-side-content")) {
+            document.querySelector('body').classList.toggle('hide-sub-sidebar');
+        }
+
+    }
+    @HostListener('window:scroll', [])
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-group-list-sidebar/inventory-group-list-sidebar.component.ts:22]: Remove blank line

File: apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+import { GIDDH_DATE_FORMAT, GIDDH_NEW_DATE_FORMAT_UI } from '../../../shared/helpers/defaultDateFormat';
+
+
+@Component({
+    selector: 'new-inventory-advance-search',
+    templateUrl: './new-inventory-advance-search.component.html',
+    styleUrls: ['./new-inventory-advance-search.component.scss'],
+
+})
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts:18]: Remove empty line

File: apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+        }
+    }
+    constructor(private _breakPointObservar: BreakpointObserver,
+        private generalService: GeneralService, private modalService: BsModalService){
+
+    }
+    public ngOnInit() {
+        this._breakPointObservar.observe([
+            '(max-width: 767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts:93]: this.destroyed$ is not implemented in ngOnDestroy

File: apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+import * as moment from 'moment/moment';
+import { GIDDH_DATE_FORMAT, GIDDH_NEW_DATE_FORMAT_UI } from '../../../shared/helpers/defaultDateFormat';
+
+@Component({
+    selector: 'stock-group-list',
+    templateUrl: './stock-group-list.component.html',
+    styleUrls: ['./stock-group-list.component.scss'],
+
+})
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts:14]: Remove empty lines

File: apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+        if (value && value.name) {
+            this.selectedRangeLabel = value.name;
+        }
+        this.hideGiddhDatepicker();
+        if (value && value.startDate && value.endDate) {
+            this.selectedDateRange = { startDate: moment(value.startDate), endDate: moment(value.endDate) };
+            this.selectedDateRangeUi = moment(value.startDate).format(GIDDH_NEW_DATE_FORMAT_UI) + " - " + moment(value.endDate).format(GIDDH_NEW_DATE_FORMAT_UI);
+            this.fromDate = moment(value.startDate).format(GIDDH_DATE_FORMAT);
+            this.toDate = moment(value.endDate).format(GIDDH_DATE_FORMAT);
+            // if ((this.daybookQueryRequest.from !== this.fromDate) || (this.daybookQueryRequest.to !== this.toDate)) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts:89]: Consider removing commented code

File: apps/web-giddh/src/app/new-inventory/new-inventory.component.ts [hunk 13]
```diff
@@ -0,0 +1,194 @@
...
+    */
+    public toggleSettingPane(event?): void {
+        this.toggleBodyClass();
+
+        if (this.isMobileScreen && event && this.asideInventorySidebarMenuState === 'in') {
+            this.asideInventorySidebarMenuState = "out";
+        }
+    }
+
+    constructor(
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/new-inventory.component.ts:174]: constructor should be at the top of all methods and below member variables

File: apps/web-giddh/src/app/new-inventory/new-inventory.component.ts [hunk 14]
```diff
@@ -0,0 +1,194 @@
...
+    openModal(inventoryAdvanceSearch: TemplateRef<any>) {
+        this.modalRef = this.modalService.show(inventoryAdvanceSearch,
+            Object.assign({}, { class: 'modal-lg' })
+        );
+    }
+
+    public ngOnInit() {
+        this._breakPointObservar.observe([
+            '(max-width: 767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/new-inventory.component.ts:190]: this.destroyed$ is not implemented in ngOnDestroy

File: apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.ts [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+    public isMobileScreen: boolean = true;
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    constructor(
+        private _breakPointObservar: BreakpointObserver,) {
+    }
+    public ngOnInit() {
+        /* added break point of mobile screen size  */
+        this._breakPointObservar.observe([
+            '(max-width: 767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.ts:24]: this.destroyed$ is not implemented in ngOnDestroy

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+        }
+        this.hideGiddhDatepicker();
+        if (value && value.startDate && value.endDate) {
+            this.selectedDateRange = { startDate: moment(value.startDate), endDate: moment(value.endDate) };
+            this.selectedDateRangeUi = moment(value.startDate).format(GIDDH_NEW_DATE_FORMAT_UI) + " - " + moment(value.endDate).format(GIDDH_NEW_DATE_FORMAT_UI);
+            this.fromDate = moment(value.startDate).format(GIDDH_DATE_FORMAT);
+            this.toDate = moment(value.endDate).format(GIDDH_DATE_FORMAT);
+        }
+    }
+    constructor(
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:84]: Constructor should come before other methods and after member variables

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+        private generalService: GeneralService, private modalService: BsModalService,
+        private _breakPointObservar: BreakpointObserver, ) {
+
+    }
+
+    public ngOnInit() {
+        this._breakPointObservar.observe([
+            '(max-width: 767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:93]: this.destroyed$ is not implemented in ngOnDestroy

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts [hunk 18]
```diff
@@ -0,0 +1,34 @@
...
+    constructor(
+        private _breakPointObservar: BreakpointObserver,) {
+
+    }
+
+    public ngOnInit() {
+        /* added break point of mobile screen size  */
+        this._breakPointObservar.observe([
+            '(max-width: 767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts:29]: this.destroyed$ is not implemented in ngOnDestroy

File: apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.ts [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+    /* Create combo aside pane open function */
+    public addAdjustment(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    constructor(
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.ts:39]: Constructor should come before all methods and after member variables

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts [hunk 20]
```diff
@@ -0,0 +1,108 @@
...
+    public adjustInventory(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+
+    constructor(
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts:101]: Constructor should come before all methods and after member variables

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.html [hunk 21]
```diff
@@ -0,0 +1,164 @@
+<div class="container-fluid inventory-about-page static-tabs-on-page">
+    <div class="about-group pt-15">
+        <div class="row">
+            <div class="col-10 col-md-6">
+                <div class="group-detail d-flex w-100 align-items-center">
+                    <div class="group-thumbnail mr-15">
+                        <img src="../../../../assets/images/Login-Page-Image.png" alt="group-icon">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.html:7]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.scss [hunk 22]
```diff
@@ -0,0 +1,80 @@
...
+            display: inline-flex;
+            img{
+                width: 100%;
+                height: 100%;
+            }
+        }
+    }
+}
+@media(max-width:767px){
+    // .group-detail-right-btn{
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.scss:67]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.scss [hunk 23]
```diff
@@ -0,0 +1,80 @@
...
+        }
+        .group-name{
+            h4{
+                font-family: 'opensans-semibold';
+                font-size: 16px;
+            }
+        }
+    }
+}
+.custom-tabs::ng-deep .nav-tabs{
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.scss:20]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.html [hunk 24]
```diff
@@ -0,0 +1,110 @@
+<div class="container-fluid static-tabs-on-page">
+    <div class="about-group pt-15">
+        <div class="row">
+            <div class="col-10 col-md-6">
+                <div class="group-detail d-flex w-100 align-items-center">
+                    <div class="group-thumbnail mr-15">
+                        <img src="../../../../assets/images/Login-Page-Image.png" alt="group-icon">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.html:7]: DOne

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss [hunk 25]
```diff
@@ -0,0 +1,125 @@
...
+            }
+            .group-name{
+                h4{
+                    font-size: 14px;
+                }
+            }
+        }
+    }
+}
+    .custom-tabs::ng-deep .nav-tabs{
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss:116]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html [hunk 26]
```diff
@@ -0,0 +1,605 @@
+<div class="container-fluid inventory-about-page static-tabs-on-page">
+    <div class="about-group pt-15">
+        <div class="row">
+            <div class="col-10 col-md-6">
+                <div class="group-detail d-flex w-100 align-items-center">
+                    <div class="group-thumbnail mr-15">
+                        <img src="../../../../assets/images/Login-Page-Image.png" alt="group-icon">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html:7]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html [hunk 27]
```diff
@@ -0,0 +1,605 @@
...
+                                <th width="20%">Stock Name <span class="red-text">*</span></th>
+                                <th width="20%">Unit <span class="red-text">*</span></th>
+                                <th width="16%">Qty <span class="red-text">*</span></th>
+                                <th width="20%">Selling Price</th>
+                                <th width="20%">Purchase Price</th>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td width="3%">
+                                        <img src="../../../../assets/images/add-image.svg" width="60" height="60">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html:582]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.scss [hunk 28]
```diff
@@ -0,0 +1,108 @@
...
+    .right-img-div{
+        display: none;
+    }
+    .more-add-button{
+        margin-top: 8px;
+    }
+    .select-product-type{
+        padding-bottom: 10px;
+    }
+    .custom-tabs::ng-deep .nav-tabs{
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.scss:99]: Done

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.scss:6]: Done

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts:13]: Done

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts:80]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-group/create-group.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-group/create-group.component.scss:7]: Done

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.html:45]: Done

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss:33]: Done

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss:123]: Done

File: apps/web-giddh/src/app/new-inventory/component/inventory-group-list-sidebar/inventory-group-list-sidebar.component.ts (hunk 8 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory-group-list-sidebar/inventory-group-list-sidebar.component.ts:22]: Done

File: apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts (hunk 9 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts:18]: Done

File: apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts (hunk 11 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts:14]: Done

File: apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts (hunk 12 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts:89]: REmoved

File: apps/web-giddh/src/app/new-inventory/new-inventory.component.ts (hunk 13 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/new-inventory.component.ts:174]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts (hunk 16 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:84]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts (hunk 17 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:93]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts (hunk 18 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts:29]: Done

File: apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.ts (hunk 19 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.ts:39]: Done

File: apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts (hunk 20 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.ts:101]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.ts (hunk 15 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-combo-detail/about-combo-detail.component.ts:24]: Done

File: apps/web-giddh/src/app/new-inventory/new-inventory.component.ts (hunk 14 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/new-inventory.component.ts:190]: Done

File: apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts:93]: Done

File: apps/web-giddh/src/app/app.routes.ts [hunk 29]
```diff
@@ -66,6 +68,7 @@ export const ROUTES: Routes = [
     { path: 'select-plan', component: SelectPlanComponent },
     { path: 'billing-detail', component: BillingDetailComponent },
     { path: 'billing-detail/buy-plan', component: BillingDetailComponent },
+    //{ path: 'new-inventory/create-group', component: InventoryCreateGroupComponent },
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/app.routes.ts:71]: We will reuse it in future

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.scss [hunk 30]
```diff
@@ -0,0 +1,316 @@
+@import "../../../../assets/css/color.scss";
+.create-new-inventory-page{
+    .icon-cross{
+            position: absolute;
+            right: 10px;
+            top: 3px;
+            padding: 10px;
+            z-index: 10;
+        }
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.scss:9]: format file

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss [hunk 31]
```diff
@@ -0,0 +1,173 @@
...
+}
+
+.right-side-wrapper {
+    height: 600px;
+    overflow-y: auto;
+    padding-bottom: 60px;
+}
+.add-item-btn{
+        display: flex !important;
+        flex-wrap: nowrap;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss:141]: format file

File: apps/web-giddh/src/app/new-inventory/component/inventory-sidebar/inventory-sidebar.component.html [hunk 32]
```diff
@@ -0,0 +1,37 @@
+
+    <div class="aside-pane" (clickOutside)="closeAsidePaneIfMobile($event)" asideInventorySidebarMenuState>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/inventory-sidebar/inventory-sidebar.component.html:2]: format file

File: apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts [hunk 33]
```diff
@@ -0,0 +1,94 @@
...
+    public toDate: string;
+    /* Selected range label */
+    public selectedRangeLabel: any = "";
+    /* This will store the x/y position of the field to show datepicker under it */
+    public dateFieldPosition: any = { x: 0, y: 0 };
+    @ViewChild('datepickerTemplate') public datepickerTemplate: ElementRef;
+
+    constructor(
+        private generalService: GeneralService,
+        private modalService: BsModalService,) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts:38]:
```suggestion
        private modalService: BsModalService) {
```

File: apps/web-giddh/src/app/new-inventory/new-inventory.component.scss [hunk 34]
```diff
@@ -0,0 +1,111 @@
...
+                border: none;
+            }
+             .icon-search1{
+                border-right: 1px solid $gainsboro;
+             }
+        }
+            &.full-width-search-group{
+                width: 228px;
+                .icon-search1.pos-abs {
+
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/new-inventory.component.scss:31]: format file

**Review by ravinderwalkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts [hunk 35]
```diff
@@ -0,0 +1,106 @@
...
+    public dateFieldPosition: any = { x: 0, y: 0 };
+    /* this will store image path*/
+    public imgPath: string = '';
+
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    @ViewChild('datepickerTemplate') public datepickerTemplate: ElementRef;
+
+    constructor(
+        private generalService: GeneralService, private modalService: BsModalService,
+        private _breakPointObservar: BreakpointObserver,) {
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:46]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts [hunk 36]
```diff
@@ -0,0 +1,41 @@
...
+export class AboutProductServiceDetailComponent implements OnInit {
+    /* This will store if device is mobile or not */
+    public isMobileScreen: boolean = true;
+    /* this will store image path*/
+    public imgPath: string = '';
+
+    public productContent: boolean = true;
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    constructor(
+        private _breakPointObservar: BreakpointObserver,) {
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts:23]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.scss (hunk 30 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.scss:9]: Done

File: apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss (hunk 31 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory-adjust-bulk-entry/inventory-adjust-bulk-entry.component.scss:141]: Done

File: apps/web-giddh/src/app/new-inventory/component/inventory-sidebar/inventory-sidebar.component.html (hunk 32 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory-sidebar/inventory-sidebar.component.html:2]: Done

File: apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts (hunk 33 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/stock-group-list/stock-group-list.component.ts:38]: Done

File: apps/web-giddh/src/app/new-inventory/new-inventory.component.scss (hunk 34 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/new-inventory.component.scss:31]: Done

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/new-inventory.component.scss:31]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts (hunk 35 above)
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:46]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts (hunk 36 above)
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.ts:23]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.scss (hunk 30 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.scss:9]: Done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**